docs/git-commands.md

```diff
@@ -0,0 +1,101 @@
+# Git Hub Command Guide
+The following is a number of scenarios that you will encounter while contributing to this repo, a convention in this document is to use `<< >>` to indicate text that you will need to replace when running the commands on your own machine.
+
+## Clone this REPO
+Before you begin, you will need to clone this repo
+```
+git clone https://github.com/MountSionCBSSecondary/SchoolWebsite.git
+```
+The above command will download a copy of the repo. This copy of the repo will have the remote name of `origin`.
+
```

Comment:
Might be worth adding `no_push` on origin

Reply:
The `no_push` command only works when adding a remote, because of the way we have this guide the `master` will be the origin remote, we also have master locked down so they should not be able to push to master.